.mvn/gradle-enterprise-custom-user-data.groovy

```diff
@@ -96,3 +96,20 @@ if (System.env.GITHUB_ACTIONS) {
     }
 }
 
+// Check current Maven version and Maven Wrapper version  alignment
+def runtimeInfo = (org.apache.maven.rtinfo.RuntimeInformation) session.lookup("org.apache.maven.rtinfo.RuntimeInformation")
+def currentMavenVersion = runtimeInfo?.getMavenVersion()
```

Comment:
Regarding variable naming: instead of `current` since `current` could be used to refer to the maven daemon version or the wrapper version, can we use `runtimeMavenVersion`?